### PR TITLE
Toolbar and Unity Resyncer

### PR DESCRIPTION
**Pull request by victorxheng**
The new Tab-Holder prefab uses a horizontal layout group and content size fitter to format the tabs and can be modified easily in the future. Each tab prefab is a button updated with a Text Mesh Pro text. Scaling was also fixed.

The new Add-Category prefab is built off what was previously there but this time uses horizontal layout groups and content size fitters to format the button icons and labels, which are now updated with Text Mesh Pro text. The scaling was also fixed.

AARD-1266:
[Remade Toolbar Tabs](https://jira.autodesk.com/browse/AARD-1266)

### CODE REVIEW

**Review by matthew17754 — APPROVED** [review]
Looks ok, other than the merge conflicts

**Comment by victorxheng** [comment]
I made toolbar switching into a prefab and UnityResyncer unit test.

[Create UI Toolbar](https://jira.autodesk.com/browse/AARD-1262)
[Redo Toolbar Tabs](https://jira.autodesk.com/browse/AARD-1266)
[Unity Resyncer Testing](https://jira.autodesk.com/browse/AARD-1264)

File: engine/Assets/Scripts/SampleTest.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+using Synthesis.Util;
+using Synthesis.UI;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class SampleTest : MonoBehaviour
```
Comment (KyroVibe) [inline engine/Assets/Scripts/SampleTest.cs:10]: Remove this Test script as it was implemented as a unit test somewhere else

**Review by KyroVibe — CHANGES_REQUESTED** [review]
Add a new gameobject with the sole intention of containing the UnityResyncerComponent

**Review by KyroVibe — APPROVED** [review]
Made some final touches with the gitignore and some files you accidently deleted from the git history. All good to go

**Pull request merged**